addons/binding/org.openhab.binding.dwdunwetter/README.md

```diff
@@ -0,0 +1,104 @@
+The DWD provides weather warnings for the german area. Regions outside of germany are not supported.
+
+## Supported Things
+
+This binding supports exactly one thing - Weather Warning. One Thing provides one or multiple warnings for an city.
+
+
+
+## Thing Configuration
+
+| Property | Default | Required | Description |
+| ---------| ------- | -------- | ----------- |
+| cellId   | -       | Yes      | Id of the area to retrieve weather warnings. For a list of valid IDs look at https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv.csv. With the % sign at the end it is possible to query multiple cells at once. For example with 8111% are cells retrieved that starts with 8111. |
+| refresh  | 30      | No       | Time between to API requests in minutes. Minimum 15 minutes. |
+| warningCount | 1   | No       | Number of warnings to provide. For each warning there will multiple channels.| 
+
+The plugin will deliver no warnings if the number of retrieved warnings for one thing is to big. This can only happen if you select to many Cells with the %-Operator. It will happen if the area contains about 300+ warnings.
+
+Example:
+
+```
+dwdunwetter:dwdwarnings:cologne "Warnings Cologne" [ cellId="105315000", refresh=15, warningCount=1 ]
+```
+
+
+## Channels
+
+The are multiple channlöes for every weather warning. The channels are numbered, so channels ending in 1 are for the first warning, ending with 2 are for the second warning and so on.
+The warnings will be sorted by severity and for warnings with same severity by the valid from date. This ensures that in the channels for the first warning will always be the highest severity.
+If the API returns more warnings than configured number in the thing, the warnings with the lowest severity will be dropped.  
+ 
+| Channel | Type | Beschreibung |
+| ------- | ---- | ------------ |
+| warningN | Switch | ON if a warning is present, OFF else |
+| UpdatedN | Trigger Channel | Triggers NEW if a warning is send the first time |
+| severityN | String | Severity of the warning. Possible values are Minor, Moderate, Severe and Extreme |
+| headlineN | String | Headline of the warning like "Amtliche Warnung vor FROST" |
+| descriptionN | String | Textual description of the warning |
+| eventN | String | Type of the warning, e.g. FROST |
+| effectiveN | DateTime | Issued Date and Time |
+| onsetN | DateTime | Start Date and Time for which the warning is valid |
+| expiresN | DateTime | End Date and Time for which the warning is valid |
+| altitudeN | Number:Length | Lower Height above sea level for which the warning is valid |
+| ceilingN | Number:Length | Upper Height above sea level for which the warning is valid |
+| urgencyN | String| Urgency of the warning Possible values are Future and Immediate |
+| instructionN | String | Additional instructions and safety informations |
```

Comment:
It's easier to review tables when they are nicely aligned.
You can for instance use the [Markdown tables generator](https://www.tablesgenerator.com/markdown_tables) for this.

Reply:
That's a nice tool, thx.

Done